tests/test_detection.py

```diff
@@ -95,10 +97,10 @@ def test_step_two(cytotable_NF1_contamination_data_df: pd.DataFrame):
     )
 
     # Execute step 1 as it is required for step 2
-    detector.step_one_test()
+    detector.check_skew_and_variable()
 
     # Execute step 2
-    detector.step_two_test()
+    detector.check_feature_means()
```

Comment:
Based on the comments above about the key and method evaluations it could be useful to consider additional scrutiny over this method (perhaps testing for some of the keys).